contracts/modules/staking/StakingPool.sol

```diff
@@ -1202,13 +1209,8 @@ contract StakingPool is IStakingPool, ERC721 {
 
     uint _activeStake = activeStake;
     uint _stakeSharesSupply = stakeSharesSupply;
-    uint transferAmount = topUpAmount;
     uint newStakeShares;
 
-    if (updatedDeposit.stakeShares != 0) {
-      transferAmount += _activeStake * initialDeposit.stakeShares / _stakeSharesSupply;
-    }
```

Comment:
This was causing an incorrect amount of nxm to be transferred from the user when extending and increasing the deposit into a tranche that also already had a user deposit. I'm not sure I fully understand the purpose of this code so please validate if this change is correct. For reference, this was causing the test `transfers correctly increased deposit amount if previous deposit exists` to fail.